docs/Basics.md

```diff
@@ -162,31 +186,15 @@ Currently it contains list of endpoint URLs with associated usernames and authen
 endpoint: "http://localhost:8080" # your endpoint URL, defaults to localhost
 ```
 
-## SSH key auth
+## SSH Authentication
 
 The SSK key needs to be RSA and in PEM format. To ensure your key is generated in a format that works you can generate with this command: `ssh-keygen -t rsa -b 4096 -C "your_email@example.com" -m 'PEM'`. This follows the [GitHub Instructions](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) with an additional flag ensuring it is the right format.
 
-## Deployment tokens
+## Deployment Tokens
 
 Sometimes you might need to deploy things from environments that don't have your private key (e.g. CI/CD services).  
 For this cases you can use deployment tokens. Here's how it works:
 
 1.  Make sure you are logged in to your Exoframe server
 2.  Generate new deployment token using `exoframe token` command
-3.  Use the new token to deploy your service without need to authenticate: `exoframe deploy -t $TOKEN`
-
-## Updating deployed project
-
-Exoframe provides a way to easily update already deployed projects.  
-This can be done by passing `--update` (or `-u`) flag to deploy command.  
-The way it works is quite simple:
-
-1.  Exoframe deploys new version of the given project
-2.  Exoframe then waits for them to start up
-3.  Exoframe removes the old running deployments for current project
-
-This can be used together with deployment tokens to achieve simple continuous deployment for your projects.
-
-### Updating the server
-
-The server can simply be updated by invoking `exoframe update server`.
+3.  Use the new token to deploy your service without need to authenticate: `exoframe deploy -t $TOKEN`
```

Comment:
Please add line break at the end.
Or even better - just run the whole file through prettier.